src/main/haskell/kore/src/Kore/Builtin/Map.hs

```diff
@@ -546,8 +547,9 @@ unifyEquals
     tools@MetadataTools { symbolOrAliasSorts }
     substitutionSimplifier
     unifyEqualsChildren
+    p1 p2
```

Comment:
Why `p1`, `p2`.  Were they used for `trace`? Does it make sense to keep them for future use?